src/paystack.tsx

```diff
@@ -81,14 +82,7 @@ const Paystack: React.ForwardRefRenderFunction<React.ReactNode, PayStackProps> =
                 ${getChannels(channels)}
                 ${refNumberString}
                 ${subAccountString}
-                metadata: {
-                custom_fields: [
-                        {
-                        display_name:  '${firstName + ' ' + lastName}',
-                        variable_name:  '${billingName}',
-                        value:''
-                        }
-                ]},
+                metadata: ${JSON.stringify(metadata)},
```

Comment:
@hurshore Since this prop is now optional, please add a fallback (something like the previous one would be great!).